src/ResourceManager/Network/Commands.Network/help/Update-AzureRmVirtualHubVnetConnection.md

```diff
@@ -0,0 +1,244 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.network/update-azurermvirtualhubvnetconnection
+schema: 2.0.0
+---
+
+# Update-AzureRmVirtualHubVnetConnection
+
+## SYNOPSIS
+Updates an existing HubVirtualNetworkConnection.
+
+## SYNTAX
+
+### ByHubVirtualNetworkConnectionName (Default)
+```
+Update-AzureRmVirtualHubVnetConnection -ResourceGroupName <String> [-ParentResourceName <String>]
```

Comment:
Can you update this again? Looks like some parameters are optional here that should not be optional.